streams/src/test/java/org/apache/kafka/streams/smoketest/BrokerCompatibilityTest.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.smoketest;
```

Comment:
This is minor, but I'm wondering if moving forward we'll continue calling this folder/package "smoketest". We'll probably end up putting in a lot of files here, like this one, to be used for system testing. Is it worth calling it "org.apache.kafka.streams.systemtest" or something like that. I realise it's not directly related to this PR, however in a sense this PR doesn't have much to do with "smoke" tests. Thanks.